e2e/path/zonefile

```diff
@@ -66,3 +60,59 @@ _redirect.host-regex.host.path.example.com.            IN TXT "v=txtv0;type=host
 chaining-regex.path.path.example.com.                  IN A 172.20.10.2
 _redirect.chaining-regex.path.path.example.com.        IN TXT "v=txtv0;type=path;to=https://fallback-to.path.path.example.com;code=302"
 _redirect._.chaining-regex.path.path.example.com.      IN TXT "v=txtv0;type=path;re=/random/;to=https://fallback.path.path.example.com/{1}"
+
+srcrecord.path.path.example.com.                            IN A 172.20.10.2
```

Comment:
Why the need for both srcrecord as well as sourcerecord etc.?
Might need better naming to declare the difference. From the current name it feels like it should be the same. One abbreviated the other one not.

Reply:
Oops, missed this one when fixed the rebase conflicts.